src/model/trust.model.ts

```diff
@@ -89,6 +89,8 @@ export interface TrustIndividual {
   ceased_date_month?: string;
   ceased_date_year?: string;
   relevant_period?: boolean;
+  usual_residential_address?: Address
```

Comment:
not sure why we need to add the address to the trusts model? it already has ura and sa address fields

Reply:
With the flag set to `true`, "ura" and "sa" fields where all resolving to `undefined` or `null`. Adding "usual_residential_address" and "service_address" field objects that map to the DB values resolved this.  The refactor now works with either value for the flag.

I could have missed something so happy to go over this with you whenever you have a few moments to spare.

Reply:
In that case, will give you a shout when you have a moment and show you the issue. Perhaps you might advise on a more suitable approach to resolve it. I could've missed something.